docs/alma_publishing_jobs_schedule.md

```diff
@@ -0,0 +1,7 @@
+# Schedule of Publishing Jobs in Alma
+
+## Incremental Job
+* The job initializes daily every 6 hours, at 4:00am, 10:00am, 4:00pm, 10:00pm. It takes around 60 min to complete and get published. 
```

Comment:
Time zone? Does it change in EDT vs. EST?

Reply:
I'll update it is Princeton time

Reply:
Sounds good, thanks!